csharp/Squidex.ClientLibrary/Squidex.ClientLibrary/SquidexOptions.cs

```diff
@@ -155,6 +156,17 @@ public IHttpClientFactory ClientFactory
             }
         }
 
+        public TimeSpan HttpClientTimeOut
+        {
+            get => httpClientTimeOut;
```

Comment:
Can you use a normal getter here, just for consistency?

Reply:
Fixed!